userspace/libsinsp/container_engine/cri.cpp

```diff
@@ -211,7 +211,7 @@ void cri::set_extra_queries(bool extra_queries) {
 bool cri::resolve(sinsp_container_manager* manager, sinsp_threadinfo* tinfo, bool query_os_for_missing_info)
 {
 	sinsp_container_info container_info;
-	sinsp_container_info *existing_container_info;
+	std::shared_ptr<sinsp_container_info> existing_container_info;
```

Comment:
everywhere you're using std::shared_ptr, you should use sinsp_container_info::mutable_ptr_t

Reply:
I think all of these go away with the next patch that makes the entries in the map const.